Parser/convertECOrawVar.m

```diff
@@ -60,9 +60,7 @@
         
     case 'CHL' %ug/l (470/695nm)
         name = 'CPHL';
-        comment = ['Artificial chlorophyll data computed from bio-optical ' ...
-            'sensor raw counts measurements. Originally expressed in ' ...
-            'ug/l, 1l = 0.001m3 was assumed.'];
+        comment = getCPHLcomment('unknown','470nm','695nm');
```

Comment:
I think in the future, it would be good not to create new branches, PR's ... when working on a similar PR. It makes it really hard to follow with everyone's comments.
For example, why is this line changed? In your previous PR, it was 
```comment = getCPHLcomment('unknown');```
https://github.com/aodn/imos-toolbox/pull/581/files#diff-32be37807d5ec8e56a7cb8fc96406b52R63

Reply:
The function to create comments need to handle other custom entires (say excitation wavelength). Rebase was the only way. GG asked me to change commit order to make sure the code is not broken. The request of linter also required a rebase.

Reply:
sorry, I don't understand what you're talking about. I'm saying that this line is 
``` comment = getCPHLcomment('unknown','470nm','695nm'); ```

while in the previous PR, it was 
``` comment = getCPHLcomment('unknown');```

Reply:
Yes, this is because Hugo did a bit more work that required ;-) and revisited the instrument documentation so that he can now specify which wavelengths are being used.

Reply:
@lbesnard,

In the first round of the implementation, I look at all the comments. Some parsers had wavelength information, some didn't. So I took the decision to remove it for consistency. In the first review, GG said that wavelength info needs to be kept. Since this parser didn't have any prev. info about the wavelengths, I had to search for it (and since I was doing it, I also double-check all other wavelength comments). 

Finally, the function was extended with new arguments to build the comment correctly and explicitly.